docs/articles/user-guide/cli-commands.md

```diff
@@ -23,31 +23,39 @@ These Salesforce CLI commands are available in Salesforce Extensions for VS Code
 - `force:config:list`: **SFDX: List All Config Variables**
+- `force:config:set --isvebuggerSid --isvDebuggerUrl --instanceUrl`: **Create and Set Up Project for ISV Debugging** (step 2)
+- `force:data:record:create --sobjecttype --values --usetoolingapi`: **Turn On Apex Debug Log for Replay Debugger**
+- `force:data:record:delete --sobjecttype --sobjectid --usetoolingapi`: **Turn Off Apex Debug Log for Replay Debugger**
 - `force:data:soql:query`: **SFDX: Execute SOQL Query with Currently Selected Text**
 - `force:data:soql:query ...`: **SFDX: Execute SOQL Query...**
 - `force:lightning:app:create ...`: **SFDX: Create Lightning App**
-- `force:lightning:component:create ...`: **SFDX: Create Lightning Component**
+- `force:lightning:component:create ...`: **SFDX: Create Lightning Web Component**
 - `force:lightning:event:create ...`: **SFDX: Create Lightning Event**
 - `force:lightning:interface:create ...`: **SFDX: Create Lightning Interface**
+- `force:mdapi:retrieve --retrievetargetdir --unpacked --targetusername`: **Create and Set Up Project for ISV Debugging** (step 4)
+- `force:mdapi:convert --rooter --outputdir force-app`: **Create and Set Up Project for ISV Debugging** (step 5)
+- `force:package:installed:list --targetusername`: **Create and Set Up Project for ISV Debugging** (step 6)
 - `force:org:create --setdefaultusername ...`: **SFDX: Create a Default Scratch Org**
-- `force:org:display`: **SFDX: Display Org Details for Default Scratch Org**
+- `force:org:display`: **SFDX: Display Org Details for Default Org**
 - `force:org:display --targetusername ...`: **SFDX: Display Org Details...**
 - `force:org:open`: **SFDX: Open Default Org**
 - `force:project:create --template standard ...`: **SFDX: Create Project**
 - `force:project:create --template standard --manifest ...`: **SFDX: Create Project with Manifest**
-- `force:source:delete`: **SFDX: Delete from Project and Org** (beta)
-- `force:source:deploy`: **SFDX: Deploy Source to Org** (beta)
-- `force:source:deploy --manifest ...`: **SFDX: Deploy Source in Manifest to Org** (beta)
+- `force:source:delete`: **SFDX: Delete from Project and Org**
+- `force:source:deploy`: **SFDX: Deploy Source to Org**
+- `force:source:deploy --manifest ...`: **SFDX: Deploy Source in Manifest to Org**
 - `force:source:pull`: **SFDX: Pull Source from Default Scratch Org**
 - `force:source:pull --forceoverwrite`: **SFDX: Pull Source from Default Scratch Org and Override Conflicts**
 - `force:source:push`: **SFDX: Push Source to Default Scratch Org**
 - `force:source:push --forceoverwrite`: **SFDX: Push Source to Default Scratch Org and Override Conflicts**
-- `force:source:retrieve`: **SFDX: Retrieve Source from Org** (beta)
-- `force:source:retrieve --manifest ...`: **SFDX: Retrieve Source in Manifest from Org** (beta)
+- `force:source:retrieve`: **SFDX: Retrieve Source from Org**
+- `force:source:retrieve --manifest ...`: **SFDX: Retrieve Source in Manifest from Org**
```

Comment:
We're missing `SFDX: Retrieve This Source from Org`

Reply:
Added the command.